src/UsgsAstroPlugin.cpp

```diff
@@ -154,12 +154,15 @@ std::string UsgsAstroPlugin::loadImageSupportData(const csm::Isd &imageSupportDa
   std::string imageFilename = imageSupportDataOriginal.filename();
   size_t lastIndex = imageFilename.find_last_of(".");
   std::string baseName = imageFilename.substr(0, lastIndex);
+  lastIndex = baseName.find_last_of("/");
```

Comment:
This part isn't going to be portable to windows. We'll have to use the dir_separator compiler flag bit

Reply:
:+1: Fixed in an inbound PR.